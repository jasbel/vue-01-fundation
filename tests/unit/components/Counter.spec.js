import { shallowMount, mount } from "@vue/test-utils";
import CounterVue from "@/components/Counter.vue";

describe("Counter component", () => {
  test("should match with snapshot", () => {
    const wrapper = shallowMount(CounterVue)
    expect(wrapper.html()).toMatchSnapshot();
  });
});
