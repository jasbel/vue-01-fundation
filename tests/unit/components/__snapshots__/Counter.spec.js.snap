// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter component should match with snapshot 1`] = `
<h1></h1>
<p>10 <sup>2</sup> = 100</p>
<button>Increase</button>
<button>decrease</button>
`;
